lib/widgets/Picker.dart

```diff
@@ -12,41 +13,110 @@ class _PickerState extends State<Picker> {
 
-  Widget _dot = new Container(
-    width: 5.0,
-    height: 5.0,
-    decoration: new BoxDecoration(shape: BoxShape.circle, color: Colors.black),
+  static Color brightPink = T.RandomRunColors.brightPink;
+  static Color lighterPink = T.RandomRunColors.lighterPink;
+
+  Widget _dot = Padding(
+    padding: EdgeInsets.only(top: T.Spacing.small),
+    child: Container(
+      width: 6.0,
+      height: 6.0,
+      decoration: BoxDecoration(
+        shape: BoxShape.circle, 
+        color: brightPink
+      )
+    )
+  );
+
+  BoxDecoration _boxOutline = BoxDecoration(
+    border: Border.all(
+      width: 2,
+      color: brightPink,
+    ),
+    borderRadius: BorderRadius.all(
+      Radius.circular(T.Spacing.micro)
+    ),
+    color: T.RandomRunColors.grayPink
+  );
+
+  TextStyle _highlightedStyle = TextStyle(
+    fontSize: T.FontSize.large,
+    fontWeight: FontWeight.w400
+  );
+
+  TextStyle _notHighlightedStyle = TextStyle(
+    color: lighterPink,
+    fontSize: T.FontSize.medium,
+    fontWeight: FontWeight.w100
+  );
+
+  TextStyle _chosenNumberStyle = TextStyle(
+    color: brightPink,
+    fontWeight: FontWeight.w500,
+    fontSize: T.FontSize.xlarge
   );
 
   @override
   Widget build(BuildContext context) {
     _initializePickers();
-    return new Center(
-      child: new Container(
-        child: Column(
-          children: <Widget>[
-            Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [_wholeNumberPicker, _dot, _decimalNumberPicker],
-            ),
-          ],
+    final theme = Theme.of(context);
+    return Flex(
+      direction: Axis.vertical,
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      children: <Widget>[
+        Container(
+          width: T.Spacing.xxlarge,
+          decoration: _boxOutline,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Theme(
+                data: theme.copyWith(
+                  accentColor: brightPink,
+                  textTheme: theme.textTheme.copyWith(
+                    headline: _highlightedStyle,
+                    body1: _notHighlightedStyle
+                  )
+                ),
+                child: _wholeNumberPicker,
+              ),
+              _dot,
+              Theme(
+                data: theme.copyWith(
+                  accentColor: brightPink,
+                  textTheme: theme.textTheme.copyWith(
+                    headline: _highlightedStyle,
+                    body1: _notHighlightedStyle
+                  )
+                ),
+                child: _decimalNumberPicker,
+              )
+            ],
+          ),
         ),
-      ),
+        Container(
+          margin: EdgeInsets.only(top: T.Spacing.medium, bottom: T.Spacing.small),
+          child: Text(
+            '$_wholeNumberValue.$_decimalValue',
+            style: _chosenNumberStyle
+          )
+        )
+      ]
```

Comment:
don't feel too strongly about this but I'm not sure if we really need this here? feels a little redundant to have the numbers appear twice - once in the picker and another here.

Reply:
updated design